pkg/mcs/resourcemanager/server/token_buckets.go

```diff
@@ -127,8 +134,10 @@ func (gts *GroupTokenBucketState) balanceSlotTokens(
 			slot = &TokenSlot{}
 			gts.tokenSlots[clientUniqueID] = slot
 			gts.clientConsumptionTokensSum = 0
+			slot.lastReqTime = now
```

Comment:
seems like slots can not be updated
```suggestion
			slot = &TokenSlot{lastReqTime: now}
			gts.tokenSlots[clientUniqueID] = slot
			gts.clientConsumptionTokensSum = 0
```